jinja/plugins.html

```diff
@@ -25,17 +25,19 @@ <h4 class="card-header juicy-peach-gradient">default.qubit</h4>
                     <div class="card-body">
                         <p class="card-text">
                             A simple state-vector qubit simulator written in Python.
-                            <br><br>A good choice for optimizations with a moderate number of qubits and parameters.
+                            <br><br>A good choice for optimizations with a moderate number of qubits and parameters, and stochastic
+                            expectation values.

```

Comment:
How about something like "or when using stochastic expectation values"? I'm worries the "and" might make it seem like you have to be in non-analytic mode.

Reply:
```suggestion
                            <br><br>A good choice for optimizations with a moderate number of qubits and parameters, or when using stochastic
                            expectation values.
```